cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -277,66 +271,59 @@ macro(configure_sdk_unix name architectures)
 
   foreach(arch ${architectures})
     if("${prefix}" STREQUAL "ANDROID")
+      # Get the prebuilt suffix to create the correct toolchain path when using the NDK
```

Comment:
This looks like something that it is worth moving into `SwiftAndroidSupport.cmake`. A function that returns a sysrtoo variable, for example. It will remove code here with the specifics, and will reduce specific paths below (`${SWIFT_ANDROID_NDK_SYSROOT}/usr/include` reads better than `${SWIFT_ANDROID_NDK_PATH}/toolchains/llvm/prebuilt/${_swift_android_prebuilt_build}/sysroot/usr/include` and will be easier to modify later, if the path changes again).

Something similar should be done with `SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_PREBUILT_PATH`. I don't know why those ended up in this file.

Reply:
These have been in this file for years, perhaps because these platform variables have always been set here, I'll look into reorganizing it.